### ISSUE TITLE
`add_shipping_info` and `add_payment_info` events not being triggered by order submit button at checkout

### ISSUE THREAD

**Issue by colmtroy** [duracelltomi/gtm4wp#314]
Hey @duracelltomi 
I see your hard at work on updates to gtm4wp - which is awesome :) 

You might have already addressed this, but I'm pretty sure [the submit event listener](https://github.com/duracelltomi/gtm4wp/blob/6f2a3f554b0d7d2c1bb3b88c5369a185f10dbe77/js/gtm4wp-woocommerce-enhanced.js#L1029) isn't firing at checkout. Most likely because of WooCommerce's checkout.js click event listeners.

In summary, I was expecting  `add_shipping_info` and `add_payment_info` events to be fired when the order submit button is fired (and your submit event listener has the necessary callbacks to make that happen - it's just the submit listener never fires).

If you need help beta testing the new version let me know!

**Comment by duracelltomi** [comment]
Hi,

I guess this will do the job.
https://github.com/duracelltomi/gtm4wp/commit/a4136e64fe5a273c6d996ff962d840e5a42bb10f

v1.20 beta1 is almost here, you can watch the release page of this repo and test it as soon as it is released. I would be more than happy if you could provide feedback :-)
